src/components/consul/detail/ConsulVideo.js

```diff
@@ -24,14 +24,14 @@ export const ConsulVideoComponent = ({ videoUrl }) => {
   );
 };
 
-ConsulVideoComponent.propTypes = {
+ConsulVideo.propTypes = {
   videoUrl: PropTypes.string
 };
 
 const styles = StyleSheet.create({
   videoView: {
     flex: 1,
-    width: '100%',
-    height: 250
+    height: 250,
```

Comment:
nothing new in code here, but with seeing it i wonder if `250` always works for different devices and orientations? can you have a look in different situations please.

Reply:
I tested. There is no big difference in devices of different sizes. How do you think I can do this?

Reply:
i thought for landscape for example the video will be very wide but still 250 in height. but maybe our orientation wrapper already saves us for having such "weird" renderings.

Reply:
Most probably. Should I change this?

Reply:
did you encounter problems now that need to be changed? which one?

Reply:
i didn't see any problem.

Reply:
ok, then we do not need to change anything.